playwright.config.ts

```diff
@@ -0,0 +1,81 @@
+import { defineConfig, devices } from '@playwright/test';
+
+// purposefully not using ??: we want to replace empty empty string with default
+const BASE_URL = process.env.BASE_URL || 'http://localhost:3000';
+
+let addlConfig = {};
+if (BASE_URL.includes('localhost') || BASE_URL.includes('127.0.0.1')) {
```

Comment:
Array#includes

Reply:
i don't know what this comment means? that's [String#includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) so idk where i could use an array here

unless you want
```typescript
if (['localhost', '127.0.0.1'].some((host: string) => BASE_URL.includes(host)) {
```

but i don't really like that for just two elements :thinking:

Reply:
that also words and is more dry

Reply:
i think i'm still leaning towards two-includes over array-some --- if i thought there was a chance we'd ever add a third thing i'd definitely do array-some, but i really don't think there is. two-includes fits all on one line neatly so there's not really a big maintenance/readability benefit right now either, so it kind of feels like the worse performance (small tho it is) doesn't get us much